node/services/service.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"github.com/celestiaorg/celestia-node/service/block"
+
+	ipld "github.com/ipfs/go-ipld-format"
+	"go.uber.org/fx"
+)
+
+// Components wraps services creation
+func Components() fx.Option {
+	return fx.Options(
+		fx.Provide(func(params blockServiceParams) *block.Service {
+			service := block.NewBlockService(params.Fetcher, params.Store)
+			params.Lc.Append(fx.Hook{
+				OnStart: service.Start,
+				OnStop:  service.Stop,
+			})
+			return service
+		}),
+	)
+}
+
+type blockServiceParams struct {
+	fx.In
+
+	Lc      fx.Lifecycle
+	Fetcher block.Fetcher
+	Store   ipld.DAGService
+}
```

Comment:
@vgonkivs, Now, as there are only 3 items, it doesn't make sense to extract this into separate Params struct and you can pass those directly.

Reply:
@Wondertan , done